c2corg_api/views/user.py

```diff
@@ -162,7 +168,9 @@ class UserValidateNewPasswordRest(object):
     def __init__(self, request):
         self.request = request
 
-    @json_view(validators=[validate_user_from_nonce, validate_json_password])
+    @json_view(validators=[
```

Comment:
Resources for validating a nonce received by email are public thus should not use `@restricted_view`.